launch_testing/launch_testing/loader.py

```diff
@@ -93,6 +93,13 @@ def __init__(self,
         self.pre_shutdown_tests = pre_shutdown_tests
         self.post_shutdown_tests = post_shutdown_tests
 
+        #  Duration for which the ReadyToTest action waits for the processes
+        #  to start up, 15 seconds by default
+        self.timeout = None
```

Comment:
you could use `15` here directly, and avoid the `timeout if timeout not None else 15` logic elsewhere

Reply:
I see, makes sense.